viz.yaml

```diff
@@ -170,7 +170,7 @@ visualize:
   -
     id: futureSuitabilityFig
     location: figures/futureSuitabilityFig.svg
-    depends: processedFutureSuitability
+    depends: [processedFutureSuitability, siteTextData]
```

Comment:
hmm...it works as is, but the viz function reads in siteText internally. Agreed that it would be better to specify it. Good idea.